client/src/app/core/auth/pure-auth-user.model.ts

```diff
@@ -0,0 +1,156 @@
+// pure version of auth-user, that doesn't import rxjs packages
+import { User } from '@app/core/users/user.model'
```

Comment:
`pure-auth-user.model.ts` should not be in the angular app so the embed does not depend on the `app` directory 

And so you should not import `User` from the angular app either